ArduPlane/tailsitter.cpp

```diff
@@ -562,6 +562,9 @@ bool Tailsitter::transition_vtol_complete(void) const
     const float trans_angle = get_transition_angle_vtol();
     if (labs(plane.ahrs.pitch_sensor) > trans_angle*100) {
         gcs().send_text(MAV_SEVERITY_INFO, "Transition VTOL done");
+        // clear inverted flight flag to make behaviour consistent
+        // with other quadplane types
+        plane.inverted_flight = false;
```

Comment:
There are lots of other cases that return true from this function. You could put it here:

https://github.com/ArduPilot/ardupilot/blob/f328a659ef6a0acea80c7fb16c2c621fb1f3be15/ArduPlane/tailsitter.cpp#L900

I do think its OK to leave the inverted bit set, it should work as expected for tailsittters, just its inconsistent with normal quadplanes.

Reply:
fixed